pkg/util/mount/mount.go

```diff
@@ -245,12 +245,9 @@ func GetDeviceNameFromMount(mounter Interface, mountPath string) (string, int, e
 	return device, refCount, nil
 }
 
-// IsNotMountPoint determines if a directory is a mountpoint.
-// It should return ErrNotExist when the directory does not exist.
-// This method uses the List() of all mountpoints
-// It is more extensive than IsLikelyNotMountPoint
-// and it detects bind mounts in linux
```

Comment:
I removed these comments because this interface has already been described in detail [here](https://github.com/kubernetes/kubernetes/blob/0045c374c2f9f201683e1c1a3ff706784b99daef/pkg/util/mount/mount.go#L53-L60).